app/routes/task.py

```diff
@@ -0,0 +1,184 @@
+
+    return model
+
+def validate_request(cls,data):
+    try:
+        new_cls = cls(title =data["title"],
+                description =data["description"])
+    except:
+        abort(make_response(
+            {"details": "Invalid data"}, 400))
+    return new_cls
+
+@bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    new_task = validate_request(Task,request_body)
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    return make_response({
+            "task": {
+            "id":new_task.task_id,
+            "title": new_task.title,
+            "description": new_task.description,
+            "is_complete": bool(new_task.completed_at)
+        }}
+    , 201)
+
+@bp.route("", methods=["GET"])
+def read_all_tasks():
+
+    # get the sort parameter from request
+    sort= request.args.get('sort')
+    tasks = Task.query.all()
+
+    # reverse is set to a boolean that sort equals "desc" is consider True. If it doesn't equal "desc" it False.
+    reverse = sort == "desc"
+
+    def sorting(task):
+        return task.title
+
+    tasks.sort(reverse=reverse, key=sorting)
+
+    get_response = []
+    for task in tasks:
+        get_response.append(dict(
+            id=task.task_id,
+            title=task.title,
+            description=task.description,
+            is_complete=bool(task.completed_at)
+        ))
+        
+    return jsonify(get_response)
+
+@bp.route("/<task_id>", methods=["GET"])
+def handle_task(task_id):
+
+    task = validate_model(Task,task_id)
+
+
+    get_response ={
+        f"task": {
+            "id": task.task_id,
+            "title": task.title,
+            "description": task.description,
+            "is_complete":  bool(task.completed_at)
+        }}
+
+    return get_response, 200
+
+    
+    
+
+@bp.route("/<task_id>", methods=["PUT"])
+def update_task(task_id):
+    task = validate_model(Task, task_id)
+
+    request_body = request.get_json()
+
+    task.title = request_body["title"]
+    task.description = request_body["description"]
+
+    db.session.commit()
+
+    update_response = {
+        "task": {
+            "id": task.task_id,
+            "title": task.title,
+            "description": task.description,
+            "is_complete": bool(task.completed_at)
+        }
+    }
+
+    return make_response(update_response), 200
+
+@bp.route("/<task_id>", methods=["DELETE"])
+def delete_task(task_id):
+
+    task = validate_model(Task, task_id)
+
+    db.session.delete(task)
+    db.session.commit()
+
+    task_response =  {
+        "details": f'Task {task_id} "{task.title}" successfully deleted'
+    }
+    return make_response(task_response, 200)
+
+def slack_bot(task):
+    url = "https://slack.com/api/chat.postMessage"
+    SLACK_API_TOKEN = os.environ.get("SLACK_API_TOKEN")
+
+    data= {"channel":"task-notifications", "text":f"Someone just completed the task {task.title}"}
+    headers = {"Authorization": SLACK_API_TOKEN}
```

Comment:
Looking at this value obtained from the environment, I suspect your file would contain something like `SLACK_API_TOKEN=Bearer xoxb-...` however I'd recommend using an f-string to add the "Bearer " text so that in case the API token needs to be included elsewhere without the "Bearer " prepended to it, it will be reusable.